stop-a-node.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Stop a Node
+toc: false
+---
+
+To stop a CockroachDB node, run the `cockroach quit` command with appropriate flags. This allows in-flight requests to complete and then shuts down the node.
+
+Once a node has been offline for approximately 5 minutes, CockroachDB automatically rebalances replicas from the missing node, using unaffected replicas on other nodes as sources. 
+
+<div id="toc"></div>
+
+## Synopsis
+
+~~~ shell
+# Run the command:
+$ ./cockroach quit <flags>
+
+# View help directly in your shell:
+$ ./cockroach quit --help
+~~~
+
+## Flags
+
+The `quit` command supports the following flags as well as [global flags](cockroach-commands.html#global-flags) that can be set on any command.
+
+
+Flag | Description 
+-----|------------
+`--certs` | The path to the directory containing the node's [security certificates](create-security-certificates.html). If the node was started with security (i.e., without the `--insecure` flag), the `--certs` flag is required. <br><br> **Default:** certs
+`--host` | A valid address for reaching the node. If the node advertises itself on multiple addresses (internal hostname, internal ip, external hostname, external ip, etc.), this can be any one of them. <br><br>**Default:** localhost
+`--insecure` | Whether or not the cluster is secure (authentication and encrypted client/node and inter-node communication). If the cluster is secure, set the `--certs` flag but leave this flag out. If the cluster is insecure, set this flag.
```

Comment:
I believe we can omit the `--insecure` flag here (as soon as @mberhault finishes his change anyway – Marc, you should probably remove the inclusion of the `--insecure` flag in cli/flags.go for quit as part of your security change).